### DISCUSSION TITLE
Disabling Cursor Movement.

### DISCUSSION THREAD

**Discussion by cowboy8625** [bevyengine/bevy#2756] [Q&A] [answered]
I am searching for the proper way of disabling the cursor movement for use with the [bevy_mod_picking](https://crates.io/crates/bevy_mod_picking).    I just need to set the cursor to the center of the screen and disable it  from being moved other wise I will need to fork this plugin. Unless I am missing something.

**Comment by DJMcNab** [comment, marked as answer]
Seems to be a duplicate of https://github.com/bevyengine/bevy/issues/2645

**Comment by cowboy8625** [comment]
Yup It sure is.    Dang it.
